src/components/Field/Field.test.js

```diff
@@ -153,29 +156,109 @@ describe('water-mode class', () => {
 })
 
 describe('isInHoverRange', () => {
-  describe('plot is not in hover range', () => {
-    test('returns false', () => {
+  test('indicated plot is not in hover range', () => {
+    expect(
+      isInHoverRange({
+        hoveredPlotRangeSize: 2,
+        hoveredPlot: { x: 1, y: 1 },
+        itemsSold: {},
+        x: 4,
+        y: 4,
+      })
+    ).toBe(false)
+  })
+
+  test('indicated plot is in hover range', () => {
+    expect(
+      isInHoverRange({
+        hoveredPlotRangeSize: 2,
+        hoveredPlot: { x: 1, y: 1 },
+        itemsSold: {},
+        x: 0,
+        y: 0,
+      })
+    ).toBe(true)
+  })
+
+  describe('fieldMode === OBSERVE', () => {
```

Comment:
can these instead be worded how the player would be thinking. i.e. maybe `describe("when observing the field")` and for the sprinkler one maybe `describe("when placing a sprinkler")`?

Reply:
Good idea! this is done in https://github.com/jeremyckahn/farmhand/commit/a23362cfe63e953c00b8a2ba7dff628ee2d72b21.

```
  isInHoverRange
    ✓ indicates when plot is not in hover range
    ✓ indicates when plot is in hover range (1 ms)
    when observing the field
      ✓ indicates when plot is not in hover range
      ✓ indicates when plot is in hover range
    when placing a sprinkler
      ✓ indicates when plot is not in hover range (1 ms)
      ✓ indicates when plot is in hover range
    when placing a scarecrow
      ✓ indicates that all plots are in hover range
```

This is much clearer! :+1: